### ISSUE TITLE
Missing nuisance regressors

### ISSUE THREAD

**Issue by mnarayan** [FCP-INDI/C-PAC#705]
I've been using the latest 1.0.2beta release and find that compcor regressors are not being stored in `func/functional_nuisance_regressors.mat`. 

I can confirm from my yaml configuration that 
a) compcor is turned on
b) number of components is 5. 

The output folders are named thus
`cpac/output/sym_links/pipeline_nov17__nuisance/_compcor_ncomponents_5_linear1.motion1.quadratic1.compcor1.csf1/sub-<SUBID>_ses-1/scan_task-<TASKNAME>/func/`

All the other nuisance regressors for my pipeline are present however. 
```
>> load func/functional_nuisance_regressors.mat
>> who

Your variables are:

constant   csf        linear     motion     quadratic
```

I did not have this problem in 1.0.1a release from March, 2017. 

```
>> load func/functional_nuisance_regressors.mat
>> who

Your variables are:

compcor    constant   csf        linear     motion     quadratic
```

This makes me suspicious as to whether compcor factors are being regressed out of the data at all!

**Comment by sgiavasis** [comment]
Hi @mnarayan ,

Thank you for bringing this up! We will look into this and address it in our upcoming release. More info will be posted here.

**Comment by sgiavasis** [comment]
Yup, found it. A check that was supposed to warn about missing CSF/WM masks was erroneously stopping CompCor altogether. Nice catch, thank you for pointing it out!!

Updating our tests so that won't happen going forward.

The fix will be in version 1.0.3, coming out within a few days.

**Comment by sgiavasis** [comment]
@mnarayan - We bumped this up from 1.0.3- this fix is available in the Master branch effective immediately- v1.0.2a hotfix.
